### PR TITLE
Fix deprecation warnings

### LINKED ISSUE #208: Deprecation messages
Here's the deprecation warnings I could find in the email failures:

Many of these. Latest Ansible is 2.7 so these are already removed in a released version.
> [DEPRECATION WARNING]: state=running is deprecated. Please use state=started. This feature will be removed in version 2.7.

> [DEPRECATION WARNING]: [defaults]hostfile option, The key is misleading as it can also be a list of hosts, a directory or a list of paths , use [defaults] inventory=/path/to/file|dir instead. This feature will be removed in version 2.8.

> [DEPRECATION WARNING]: 'include' for playbook includes. You should use 'import_playbook' instead. This feature will be removed in version 2.8.

Related:
> [DEPRECATION WARNING]: include is kept for backwards compatibility but usage is discouraged. The module documentation details page may explain more about this rationale.. This feature will be removed in a future release.

> [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using `result|changed` instead use `result is changed`. This feature will be removed in version 2.9.

### PR DESCRIPTION
**Pull request by seankelly**
Fixes #208.

### CODE REVIEW

File: roles/base/templates/run-ansible-pull.sh.j2 [hunk 1]
```diff
@@ -38,26 +39,35 @@ while [[ $RETRY -lt $RETRIES ]]; do
...
 done
 
-if [ $RESULT -ne 0 ]; then
-    cat $OUTPUT
+if [[ $RESULT -eq 0 ]]; then
+    # If everything was okay, check for deprecations but only at the 00:* run.
+    # These deprecations will happen on every run so only notify once a day.
+    # The -A3 seems to be enough to get all/most of the deprecation warning to
+    # make sense of what to fix.
+    hour=$(date +%H)
```
Comment (djmitche) [inline roles/base/templates/run-ansible-pull.sh.j2]: It likely won't be :00 anymore when the run is complete, right?  Perhaps capture this value at the very start of the script instead?

Comment (seankelly) [reply roles/base/templates/run-ansible-pull.sh.j2]: Hour, not minute. All of the `ansible-pull` runs are scattered across the hour so unlikely any of them start at :00. There is the risk that something will start at e.g. 00:59 and finish in the next hour, but my guess was the 11:59 run will be used for this instead.

Comment (seankelly) [reply roles/base/templates/run-ansible-pull.sh.j2]: Oh, but the variable case changes. Oops.

Comment (djmitche) [reply roles/base/templates/run-ansible-pull.sh.j2]: Haha, well, we both made some silly errors here :)

**Pull request merged**